docs/src/active-layer-transport.md

```diff
@@ -0,0 +1,23 @@
+# Active Layer Transport
+
+The following is inspired on well-known **active layer** approaches in river bed sediment transport.
+
+We suppose that loose sediment, either fresh production or disintegrated older sediment, is being transported in a layer on top of the sea bed. The flux in this layer is assumed to be directly proportional to the local slope of the sea bed $\partial_x \eta$.
+
+The active layer now contains a concentration $C_f$ particles of different grain size (for each facies $f$). If needed, $C_f = \alpha_f P_f$ where $\alpha_f$ is some facies parameter determining the fraction of production that is available for transport. The flux between two cells is then given as
+
+$$q = -\nu_f C_f {{\partial \Sum_f \eta_f} \over {\partial x}}.$$
+
+The flux again measures the change in height as contributed by each facies type. The following is a mass balance:
+
+$$\sigma + \Sum_f {{\partial \eta_f} \over {\partial t}} = -\Sum_f {{\partial q_f} \over {\partial x}} + P_f,$$
+
+where $\sigma$ is the subsidence rate. In other approaches to active layer transport there would be a factor $1/C_f$. Here we have a different interpretation: the sediment settles down after transport, such that the concentration has no impact on the change in sediment surface elevation.
+
+Combining these equations, and ignoring subsidence for the moment, we get a component-wise diffusion equation
+
+$$\partial_t \eta_f(x) = \partial_x (\nu_f \alpha_f P_f(x) \partial_x \eta_{*}(x)) + P_f(x),$$
+
+where $\eta_{*} = \sum_f \eta_f$ is the total sediment surface elevation.
+
+In our model we need to solve this equation one time-step each iteration. If we solve this using forward methods, we should be reminded of the CFL limit for diffusion equations (depending on the diffusion constants and grid size we shouldn't pick the time steps too large). Alternatively, for these two-dimensional situations, an implicit approach is feasible.
```

Comment:
@EmiliaJarochowska this is the plan I came up with now. If you're good with this, I can start implementation.

Reply:
Thanks a lot for this! Please give me until Wednesday to answer, I would like to discuss with Peter.

Reply:
We would like to check two things before proceeding:
1. What are the causes of the pinnacle-formation? We already assumed they are the outcome of the critical value for shear stress not being applicable to time steps of the order we use, but it would be prudent to investigate this
2. A concern for this approach is that it does not allow for event sedimentation (something I need to seek a definition for). But that is already partly built in into a fixed concept of active layer. For example, transport might be triggered by intensive production that leads to steepening on the slope. This approach cannot(?) capture this situation. That is what we should have seen in the previous approach but somehow it didn't happen. So I am not sure if the "pinnacles" were not an artifact of the CA. I do wonder if the two approaches could be combined, e.g. shear stressed based transport allows for more "event" layers, but then an event layer can be subject to diffusion-based transport as you proposed. This might be a computational overkill though? @Burgesski @xyl96 please comment if I missed anything

Reply:
1. On the pinnacles: I think the Warlich model confuses integrated (or event-based if you like) approaches with a differential scheme. There is in the thesis text no discussion about model stability and such. If there is more literature about this type of critical shear stress approach I would love to become the wiser.
2. Indeed events are not really a thing in the current model. Still, event driven output can still be interpreted on the basis that the CA emits discrete events. We can run any number of production+transport cycles for every CA event to boost that signal.

I'm not sure that I completely understand your point about intensive production leading to steep slopes. What aspect isn't captured by the active layer approach? I suggest implementing this model and build this scenario as a test case for validation.

The advantages of this approach:
- simple
- physically based
- basis in existing literature
- easy to implement (at least in forward Euler)
- few parameters

Reply:
There is indeed more literature but it's not always immediately obvious if it's applicable. I agree with your comments on the approach in CARBONATE3D. @Burgesski pointed out that shear-stress based approach has been successfully used in other implementations of CarboCAT, for example this one: https://github.com/Isabelle16/carbocat2018a/ giving very realistic results. I am now trying to understand what makes it work there but not for us. 

I see you have practically implemented this already so I am not going to argue. As long as it is modular, it can always be swapped if needed. As for the rest, I need to read more. TBC.